<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1096</width>
    <height>822</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Что-то</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(230, 230, 230);</string>
   </property>
   <widget class="QGroupBox" name="groupbox">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>30</y>
      <width>350</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(208, 208, 208);</string>
    </property>
    <property name="title">
     <string>InformationRecording</string>
    </property>
    <widget class="QLabel" name="labelFIO">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>25</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(50, 50, 50);</string>
     </property>
     <property name="text">
      <string>FIO</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelCourse">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>75</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(50, 50, 50);</string>
     </property>
     <property name="text">
      <string>Choose a course</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelDirection">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>160</y>
       <width>220</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(50, 50, 50);</string>
     </property>
     <property name="text">
      <string>Choose a direction</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_saveInfo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>160</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(50, 50, 50);
background-color: rgb(220, 220, 220);
</string>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinboxCourse">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>75</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(50, 50, 50);
background-color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboboxDirection">
     <property name="geometry">
      <rect>
       <x>242</x>
       <y>160</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(50, 50, 50);
background-color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_openInfo">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>220</y>
       <width>160</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(50, 50, 50);
background-color: rgb(220, 220, 220);
</string>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineeditFIO">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>25</y>
       <width>250</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSliderCourse">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>321</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">selection-color: rgb(0, 170, 0);
color: rgb(0, 170, 0);
</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widgetcalculator" native="true">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>300</width>
      <height>400</height>
     </rect>
    </property>
    <widget class="QPushButton" name="btn_two">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>255</y>
       <width>75</width>
       <height>75</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(85, 85, 85);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_three">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>255</y>
       <width>75</width>
       <height>75</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(85, 85, 85);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_one">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>255</y>
       <width>75</width>
       <height>75</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(85, 85, 85);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_multiplication">
     <property name="geometry">
      <rect>
       <x>225</x>
       <y>255</y>
       <width>75</width>
       <height>75</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(94, 94, 94);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>*</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_zero">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>330</y>
       <width>112</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(77, 77, 77);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_print_result">
     <property name="geometry">
      <rect>
       <x>112</x>
       <y>330</y>
       <width>113</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 170, 0);</string>
     </property>
     <property name="text">
      <string>=</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_division">
     <property name="geometry">
      <rect>
       <x>225</x>
       <y>330</y>
       <width>75</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(94, 94, 94);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>/</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_four">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>180</y>
       <width>75</width>
       <height>75</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(85, 85, 85);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_five">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>180</y>
       <width>75</width>
       <height>75</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(85, 85, 85);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_six">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>180</y>
       <width>75</width>
       <height>75</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(85, 85, 85);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_minus">
     <property name="geometry">
      <rect>
       <x>225</x>
       <y>180</y>
       <width>75</width>
       <height>75</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(94, 94, 94);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_seven">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>105</y>
       <width>75</width>
       <height>75</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(85, 85, 85);</string>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_eight">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>105</y>
       <width>75</width>
       <height>75</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(85, 85, 85);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_nine">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>105</y>
       <width>75</width>
       <height>75</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(85, 85, 85);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>9</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_plus">
     <property name="geometry">
      <rect>
       <x>225</x>
       <y>105</y>
       <width>75</width>
       <height>75</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(94, 94, 94);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_delete">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>65</y>
       <width>302</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>12</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(20, 20, 20);
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_result">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>300</width>
       <height>65</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(18, 18, 18);</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="margin">
      <number>11</number>
     </property>
     <property name="indent">
      <number>10</number>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_picture1">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>340</y>
      <width>261</width>
      <height>261</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_btn" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>490</y>
      <width>371</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openwindow_btn">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>40</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>CLICK</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>260</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Show password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>220</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>password123</string>
    </property>
   </widget>
   <widget class="QSlider" name="Slider">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>490</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>540</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>340</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>380</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1096</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="settingsmenu">
    <property name="title">
     <string>Настройки</string>
    </property>
    <widget class="QMenu" name="changecolormenu">
     <property name="title">
      <string>Поменять цвет фона</string>
     </property>
     <addaction name="separator"/>
     <addaction name="lightaction"/>
     <addaction name="darkaction"/>
    </widget>
    <addaction name="changecolormenu"/>
   </widget>
   <addaction name="settingsmenu"/>
  </widget>
  <action name="lightaction">
   <property name="text">
    <string>Светлый</string>
   </property>
  </action>
  <action name="darkaction">
   <property name="text">
    <string>Темный</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
